src/azure-cli/azure/cli/command_modules/config/_help.py

```diff
@@ -53,3 +53,42 @@
   - name: Unset the configuration of key `core.no_color`.
     text: az config unset core.no_color
 """
+
+helps['config parampersist'] = """
+type: group
+short-summary: Manage parameter persist
```

Comment:
We usually end the sentence with a period. 
```suggestion
short-summary: Manage parameter persist.
```

Reply:
updated